ocs_ci/ocs/node.py

```diff
@@ -908,6 +914,106 @@ def delete_and_create_osd_node_vsphere_upi(osd_node_name, use_existing_node=Fals
     return new_node_name
 
 
+def delete_and_create_osd_node_vsphere_lso(osd_node_name, use_existing_node=False):
+    """
+    Unschedule, drain and delete osd node, and creating a new osd node.
+    At the end of the function there should be the same number of osd nodes as
+    it was in the beginning, and also ceph health should be OK.
+    This function is for vSphere UPI.
+
+    Args:
+        osd_node_name (str): the name of the osd node
+        use_existing_node (bool): If False, create a new node and label it.
+            If True, use an existing node to replace the deleted node
```

Comment:
Where is use_existing_node=True covered for LSO? Is it not required?

Reply:
We have an issue to create a new worker node on vSphere from the cli before, so I used an existing worker node. Now Vijay fixed the issue, so I can remove this option if it's not necessary.